### PR TITLE
doc: Updating basic commands to up to date state

### PR DESCRIPTION
**Pull request by evilcel3ri**

### CODE REVIEW

File: src/basic_commands/dietline.md [hunk 1]
```diff
@@ -48,7 +48,7 @@ By default dietline mode is compatible with readline Emacs-like mode key binding
 
 # Vi mode
 
-Rizin also comes with in vi mode that can be enabled by toggling `scr.prompt.vi`. The various keybindings available in this mode are:
+Rizin also comes with in vi mode that can be enabled by toggling `scr.prompt.vi` (`e scr.prompt.vi=true`). The various keybindings available in this mode are:
```
Comment (wargio) [inline src/basic_commands/dietline.md:51]:
```suggestion
Rizin also comes with in vi mode that can be enabled by toggling `e scr.prompt.vi=true`. The various keybindings available in this mode are:
```

File: src/basic_commands/flags.md [hunk 2]
```diff
@@ -1,58 +1,69 @@
...
+|   0 * registers.mmio
+|  91 * relocs
+|  13 * sections
+|   3 * segments
+|  95 * strings
+|  55 * symbols
 ```
 
-Here there are some command examples:
+Here are some command examples:
```
Comment (wargio) [inline src/basic_commands/flags.md:45]:
```suggestion
Here you can find some command examples using flagspaces:
```

File: src/basic_commands/zoom.md [hunk 3]
```diff
@@ -1,82 +0,0 @@
-## Zoom
```
Comment (wargio) [inline src/basic_commands/zoom.md:1]: this was deleted, be sure to remove the link also from the book.

**Review by wargio — APPROVED** [review]

**Review by wargio — CHANGES_REQUESTED** [review]
see comments.

File: src/basic_commands/zoom.md (hunk 3 above)
Comment (evilcel3ri) [reply src/basic_commands/zoom.md:1]: done

File: src/basic_commands/sdb.md [hunk 4]
```diff
@@ -81,7 +81,7 @@ $ gcc test.c -o test
 
 ```
 $ rizin -A ./test
-[0x08048320]> k **
+[0x08048320]> k ** # list namespaces under analysis
```
Comment (wargio) [inline src/basic_commands/sdb.md:84]:
```suggestion
[0x08048320]> # list namespaces under analysis
[0x08048320]> k **
```

**Review by XVilka — APPROVED** [review]

**Pull request merged**